src/initializers/bull/bull.ts

```diff
@@ -52,8 +53,8 @@ export default class Bull {
     }
     try {
       this.metrics = {
-        depth: this.prometheus.registerGauge('bull_queue_depth', 'Bull Jobs in Queue', ['queue']),
-        failed: this.prometheus.registerGauge('bull_queue_failed', 'Bull Jobs Failed', ['queue'])
+        depth: this.prometheus.registerGauge('external', 'bull_queue_depth', 'Bull Jobs in Queue', ['queue']),
```

Comment:
❓ this means we support only one queue? We might want to label queue with queue name

Reply:
@jgerbe 
`queue` in this context is the queue name:
You can chakout the bull example, which produces something like:
```
# HELP external_bull_example_bull_queue_depth Bull Jobs in Queue
# TYPE external_bull_example_bull_queue_depth gauge
external_bull_example_bull_queue_depth{queue="queue_one"} 100
external_bull_example_bull_queue_depth{queue="queue_two"} 0

# HELP external_bull_example_bull_queue_failed Bull Jobs Failed
# TYPE external_bull_example_bull_queue_failed gauge
external_bull_example_bull_queue_failed{queue="queue_one"} 0
external_bull_example_bull_queue_failed{queue="queue_two"} 46
```

Reply:
dont be confused with the names here:
 prefix _ appName _metric
`external` _ `bull_example` _ `bull_queue_depth`